model/model_training/configs/config.yaml

```diff
@@ -54,10 +54,11 @@ defaults:
   samples_mixing: false # uses collator that mixes samples in the batch to create a single sample with possible multiple tasks within
   verbose: false
   output_dir: saved_model
-  use_custom_sampler: false
+  use_custom_sampler: true
```

Comment:
as long as the custom sampler is broken (e.g. doesn't use all samples if only single dataset is active) I would rather keep it disabled by default.